Bio/PDB/SASA.py

```diff
@@ -162,7 +162,9 @@ def compute(self, entity, level="A"):
         Example:
 
         >>> from Bio.PDB import PDBParser
+        >>> from Bio.PDB import SASA
```

Comment:
This needs to be ``from Bio.PDB.SASA import ShrakeRupley`` unless you want to change the line below to use ``SASA.ShrakeRupley``?

(Note - I say that without testing either, but I would be very surprised if the update example works as is)

Reply:
You're clearly correct. And I'm a bit embarrassed to have a cut-and-paste error on something this small. My original code had the below line saying SASA.ShrakeRupley but I think people are more likely to want it the way you suggest. This is my first time handling all of this through the web browser, but I think I've fixed it?

My apologies for causing this much chatter over a small fix.

Reply:
It's a learning experience. I still sometimes make a mistake and misjudge something as an easy easy job via the web interface, and it turns out it would have been quicker at the command line with git and my usual editor.